skyportal/tests/api/test_comments_on_spectrum.py

```diff
@@ -290,33 +289,57 @@ def test_fetch_all_spectrum_comments(
+        "name": "ampel_test.json",
+    }
+
     status, data = api(
         'POST',
         f'spectra/{spectrum_id}/comments',
         data={
-            'obj_id': public_source.id,
-            'spectrum_id': spectrum_id,
-            'text': comment_text,
+            'text': 'Looks like Ia',
             'group_ids': [public_group.id],
+            "attachment": payload,
         },
-        token=comment_token,
+        token=super_admin_token,
     )
     assert status == 200
+    assert data['status'] == 'success'
 
     status, data = api(
         'GET',
-        f'spectra/{spectrum_id}/comments',
-        token=upload_data_token,
+        f'spectra/{spectrum_id}/comments/{data["data"]["comment_id"]}',
+        token=super_admin_token,
+    )
+    assert status == 200
+    assert data['status'] == 'success'
+    assert data['data']['text'] == 'Looks like Ia'
+    assert data['data']['attachment_bytes'] == payload['body']
+    assert data['data']['attachment_name'] == payload['name']
+
+
+def test_fetch_all_comments_on_obj(comment_token, public_source):
+    comment_text = str(uuid.uuid4())
+    status, data = api(
+        'POST',
+        f'sources/{public_source.id}/comments',
+        data={'obj_id': public_source.id, 'text': comment_text},
```

Comment:
Don't need to provide `obj_id` in `data` anymore? It's in the path.

Reply:
Yes that's right. We'll clean that up in the other tests as well...